frontend/src/metabase-lib/lib/metadata/Field.ts

```diff
@@ -217,6 +219,14 @@ class FieldInner extends Base {
     return getIconForField(this);
   }
 
+  getId(): number {
```

Comment:
I'm adding this method as a way around the fact that a `field.id` might be a field ref.